assets/js/outround.js

```diff
@@ -15,27 +15,33 @@ function cycleChoice(event) {
   });
 }
 
-function populateTabCard(tabCardElement) {
-  const teamId = tabCardElement.attr("team-id");
+function populateTabCard() {
```

Comment:
```suggestion
function populateTabCards() {
```

Seems like the new approach goes from one tabcard per fn to multiple, so let's adjust the name